ush/test/README.md

```diff
@@ -0,0 +1,95 @@
+## Test bufr_query mapping, Python converter script, and ioda configuration YAML in obsForge
+This is a prototype for testing BUFR to IODA conversion and is still evolving.
+
+## Prerequisite
+- Clone and build obsForge  
+
+   ```       
+      git clone --recursive https://github.com/noaa-emc/obsForge
+   
+      cd ./obsForge/sorc/ioda
+   
+      git remote -v
+   
+      git remote set-url origin https://github.com/jcsda-internal/ioda.git
+   
+      git pull
+   
+      git checkout feature/bufr_in_parallel
+
+      cd ../spoc
+
+      git checkout feature/dump_satwind_goes
+
+      cd ../../
+   
+      ./build.sh
+   ```
+
```

Comment:
Recommend adding here a line saying to mkdir a testinput directory in ```sorc/spoc/ush/test``` and also moving your input file here. The build.sh file encodeBufr.sh does not do this. Might be worth having encodeBufr.sh doing it too.